### ISSUE TITLE
torch.nn.utils.clip_grad_norm

### ISSUE THREAD

**Issue by bmccann** [pytorch/pytorch#878]
```
>>> from torch.nn.utils import clip_grad_norm
>>> clip_grad_norm
<function clip_grad_norm at 0x7fd58b9b8d90>
```

but

```
>>> import torch
>>> torch.nn.utils.clip_grad_norm
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'torch.nn.modules.utils' has no attribute 'clip_grad_norm'
```

which is kind of confusing.

**Comment by apaszke** [comment]
It's fixed in #873.

**Comment by bmccann** [comment]
Nice
